examples/data-objects/shared-text/src/database.ts

```diff
@@ -209,22 +209,22 @@ export class GraphQLService {
                 }
             `;
 
-        const queryP = graphql<{ heroes: Hero[] }>(this.schema, query).then(
+        const queryP = graphql(this.schema, query).then(
```

Comment:
The newer 'graphql.d.ts' appears to remove the generic argument, leaving it up to the caller to cast returned values when needed.

I didn't research the motivation, but I can see an argument that <T> was not adding value.  (The T was used to perform the cast on the user's behalf...  and it's just as easy for the user to cast themselves when needed.)